packages/e2e/test/k6/endpoints/asset/get-assets.test.js

```diff
@@ -18,15 +19,17 @@ import http from 'k6/http';
  *   - 95th percentile of response time is less than 500ms
  */
 
-const ASSETS_NO_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_NO_METADATA_PER_REQUEST || 1);
-const ASSETS_ON_CHAIN_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_ON_CHAIN_METADATA_PER_REQUEST || 0);
-const ASSETS_OFF_CHAIN_METADATA_PER_REQUEST = Number.parseInt(__ENV.ASSETS_OFF_CHAIN_METADATA_PER_REQUEST || 0);
-const REQUESTS_COUNT = Number.parseInt(__ENV.REQUESTS_COUNT || 1);
-const TARGET_ENV = __ENV.TARGET_ENV || 'dev'; // only dev and live are supported
-const TARGET_NET = __ENV.TARGET_NET || 'mainnet'; // only mainnet is supported
-const supportedTargetDeployments = new Set(['dev-mainnet', 'live-mainnet']);
-const targetDeployment = `${TARGET_ENV}-${TARGET_NET}`;
-const PROVIDER_SERVER_URL = `https://${targetDeployment}.lw.iog.io`;
+// eslint-disable-next-line no-undef
+const K6_ENV = __ENV;
+
+const ASSETS_NO_METADATA_PER_REQUEST = Number.parseInt(K6_ENV.ASSETS_NO_METADATA_PER_REQUEST || 1);
+const ASSETS_ON_CHAIN_METADATA_PER_REQUEST = Number.parseInt(K6_ENV.ASSETS_ON_CHAIN_METADATA_PER_REQUEST || 0);
+const ASSETS_OFF_CHAIN_METADATA_PER_REQUEST = Number.parseInt(K6_ENV.ASSETS_OFF_CHAIN_METADATA_PER_REQUEST || 0);
+const REQUESTS_COUNT = Number.parseInt(K6_ENV.REQUESTS_COUNT || 1);
+
+const dut = k6Utils.getDut(K6_ENV, { environments: ['dev'], networks: ['mainnet'] });
+console.log(`Domain under test is: ${dut}`);
```

Comment:
I see this line is repeated somewhere else; if it was a debug line we could remove it, otherwise we could insert it into `getDut`.

Reply:
It is a valid log line, pinpointing the DUT the test ran against. I agree it should be part of `getDut` and I will move it there. Thank you!

Reply:
Done and squashed: https://github.com/input-output-hk/cardano-js-sdk/compare/d21302f4a63641388fd9122455b633db333b344d..d79ead5e6d239cb7db952d154d98ee922d16fbcc